### DISCUSSION TITLE
Issues that probably could be closed

### DISCUSSION THREAD

**Discussion by grzanka** [yaptide/ui#703] [General]
Issues:

mark tick for issues which are still present. For issues which are obsolete or solved simply close issue

@ostatni5 
- [x] https://github.com/yaptide/ui/issues/162
- [x] https://github.com/yaptide/ui/issues/539

others:
- [ ] https://github.com/yaptide/ui/issues/188
- [x] https://github.com/yaptide/ui/issues/261
- [x] https://github.com/yaptide/ui/issues/262
- [x] https://github.com/yaptide/ui/issues/305
- [ ] https://github.com/yaptide/ui/issues/306
- [x] https://github.com/yaptide/ui/issues/319
- [ ] https://github.com/yaptide/ui/issues/320
- [ ] https://github.com/yaptide/ui/issues/322
- [x] https://github.com/yaptide/ui/issues/327
- [ ] https://github.com/yaptide/ui/issues/328
- [x] https://github.com/yaptide/ui/issues/202
- [x] https://github.com/yaptide/ui/issues/295
- [x] https://github.com/yaptide/ui/issues/316

PRs:
- [x] https://github.com/yaptide/ui/pull/490

**Comment by grzanka** [comment]
@Derstilon @ostatni5 please take a look

**Reply by ostatni5** [reply]
https://github.com/yaptide/ui/issues/162 and https://github.com/yaptide/ui/issues/539 still present
